pages/index.vue

```diff
@@ -39,13 +39,7 @@
       <Cta />
       <Faq />
       <Testimonials />
-      <Cta
```

Comment:
Da haut dich adrian

Reply:
sind glaub 2 verschiedene

Reply:
Du meintest es klingt etwas "scammy".
Z.B: 
Interested about your own chat data? Reveal some interesting and entertaining facts now. Most Used Smileys, Share of speech, and more ... 

So besser? Glaub Adrian will nur 2 verschiedene

Reply:
Interested in your own chat data? Reveal some interesting and entertaining facts now. Most Used Smileys, Share of speech, and more